### PR TITLE
Frontend V2 - Style Plan Component

### PR DESCRIPTION
**Pull request by aryanshah701**
# Description

Reminder: This PR is "stacked" on top of v2-plan-component. This diff will only show styling changes.

https://user-images.githubusercontent.com/30478978/186524926-44cbf437-c298-49ac-9161-93382a4c1e31.mov

This PR styles the plan component + does some minimal styling for the home page(handles error and loading UX for the Home page, places the plan component next to a dummy sidebar). I will implement collapsing years next.

An overview of all the functionality completed and remaining for the plan component(note, there may be more stuff remaining but this is what i can think of for now).

- [x] fetch plans and populate dnd ids: previous PR
- [x] basic plan component with dnd and posting to API: previous PR
- [x] **styling: current PR**
- [ ] collapsing years
- [ ] adding a course using the add course modal
- [ ] choosing which plan to display
- [ ] choosing primary plan
  - [ ] fix backend bug of allowing users to choose a primary plan that doesn't belong to this(massive security issue) 
- [ ] validation and errors
- [ ] add class validator decorators to schedule type so that we enforce at least some shape of the schedule when posting
- [ ] figure out how to show "preview" of course in the position it was when it is being dragged around
- [ ] batch/throttle plan updates

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] This requires a run of `yarn install`

# How Has This Been Tested?

Manually tested. Check out the video.

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code where needed 
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/frontend-v2/utils/theme/index.ts [hunk 1]
```diff
@@ -72,7 +93,10 @@ const breakpoints = {
  * either direction (5xs, ..., 5xl) as needed.
  */
 const space = {
-  "2xs": "0.25rem",
+  "5xs": "0.05rem",
```
Comment (jia614) [inline packages/frontend-v2/utils/theme/index.ts:96]: :0 is all of this linda's design system stuff? :DD

File: packages/common/src/types.ts [hunk 2]
```diff
@@ -389,6 +389,14 @@ export interface Schedule2<T> {
 }
 
 export interface ScheduleYear2<T> {
+  /**
```
Comment (jia614) [inline packages/common/src/types.ts:392]: docs <3

**Review by jia614 — APPROVED** [review]
cracked-- it looks great!

File: packages/frontend-v2/utils/theme/index.ts (hunk 1 above)
Comment (aryanshah701) [reply packages/frontend-v2/utils/theme/index.ts:96]: oop not really, just values I needed to follow linda's design. i don't think linda's designs follow the design system just yet, once they do we can update it :))

File: packages/common/src/types.ts (hunk 2 above)
Comment (aryanshah701) [reply packages/common/src/types.ts:392]: AHH Yes thank you!

**Pull request closed without merging**